test/common/upstream/eds_test.cc

```diff
@@ -890,7 +894,7 @@ TEST_F(EdsTest, EndpointMovedToNewPriority) {
 TEST_F(EdsTest, EndpointMoved) {
   envoy::config::endpoint::v3::ClusterLoadAssignment cluster_load_assignment;
   cluster_load_assignment.set_cluster_name("fare");
-  resetClusterDrainOnHostRemoval();
```

Comment:
Do we still have test coverage for the `drain_on_host_removal` cases?

Reply:
Yes, EndpointRemovalClusterDrainOnHostRemoval still covers the drain on removal case.

I don't think drain_on_host_removal should have been true for this test case since, other than hiding the bug I'm fixing, it's behaviour should have been irrelevant - a host moving priority should not have had any interaction with the draining behaviour.